caffeinated/admin/extend/registration_form/Extend_Registration_Form_Admin_Page.core.php

```diff
@@ -1144,6 +1152,118 @@ protected function _update_reg_form_settings()
+                    'FHEE__Extend_Registration_Form_Admin_Page___copy_attendee_info_settings_form__form_subsections',
+                    array(
+                        'copy_attendee_info_hdr'   => new EE_Form_Section_HTML(
+                            EEH_HTML::h2(esc_html__('Copy Attendee Info Settings', 'event_espresso'))
+                        ),
+                        'copy_attendee_info' => new EE_Yes_No_Input(
+                            array(
+                                'html_label_text' => esc_html__(
+                                    'Allow copy #1 attendee info to extra attendees?',
+                                    'event_espresso'
+                                ),
+                                'html_help_text'  => esc_html__(
+                                    'Set to yes if you want to enable the copy of #1 attendee info to extra attendees at Registration Form.',
+                                    'event_espresso'
+                                ),
+                                'default'         => isset(
+                                    EE_Registry::instance()->CFG->registration->copy_attendee_info
+                                )
+                                    ? EE_Registry::instance()->CFG->registration->copy_attendee_info
+                                    : true,
+                                'required'        => false,
+                                'display_html_label_text' => false,
+                            )
+                        ),
+                    )
+                ),
+            )
+        );
+    }
+
+    /**
+     * @param EE_Registration_Config $EE_Registration_Config
+     * @return EE_Registration_Config
+     * @throws EE_Error
+     * @throws InvalidArgumentException
+     * @throws ReflectionException
+     * @throws InvalidDataTypeException
+     * @throws InvalidInterfaceException
+     */
+    public function update_copy_attendee_info_settings_form(EE_Registration_Config $EE_Registration_Config)
+    {
+        $prev_copy_attendee_info = $EE_Registration_Config->copy_attendee_info;
+        try {
+            $copy_attendee_info_settings_form = $this->_copy_attendee_info_settings_form();
+            // if not displaying a form, then check for form submission
+            if ($copy_attendee_info_settings_form->was_submitted()) {
+                // capture form data
+                $copy_attendee_info_settings_form->receive_form_submission();
+                // validate form data
+                if ($copy_attendee_info_settings_form->is_valid()) {
+                    // grab validated data from form
+                    $valid_data = $copy_attendee_info_settings_form->valid_data();
+                    if (isset($valid_data['copy_attendee_info'])) {
+                        $copy_attendee_info = $valid_data['copy_attendee_info'];
+
+                        $EE_Registration_Config->copy_attendee_info = $copy_attendee_info;
```

Comment:
see below for more info, but we need to change this to use a setter, and also don't need to capture the form value in a variable, ie:

```php
$EE_Registration_Config->setCopyAttendeeInfo( $valid_data['copy_attendee_info'] );
```